tests/e2e/rudder-npm-js/README.md

```diff
@@ -0,0 +1,26 @@
+# Getting Started with Rudder Typer
+
+## Prerequisite(If you want to use your own tracking plan)
+
+1. You have setup the tracking plan from the dashboard.
+2. Generated PAT from dashboard settings.
+3. Did the authentication step using PAT and email to generate ruddertyper.yml and plan.json file in the root level of the repo.
+4. Move ruddertyper.yml file from root level to tests/e2e/rudder-npm-js this folder.
+5. Replace the plan.json in rudder-npm-js/src/analytics with the one you generated.
+6. Delete `index.ts` and `rudder.ts` inside analytics folder.
+7. Run the command `npx rudder-typer@next`
+8. New `index.ts` and `rudder.ts` file will be generated.
+9. Add `import type { RudderAnalytics, RudderAnalyticsPreloader } from '@rudderstack/analytics-js/*';` this line at the top of rudder.ts and update rudderanalytics interface type to `rudderanalytics: RudderAnalytics | RudderAnalyticsPreloader | undefined;`
+
+## Steps to run the sample app
+
+1. From terminal go to tests/e2e/rudder-npm-js project folder.
+2. Run the command `npm i`
```

Comment:
Let's update all the dependencies to use exact versions and commit the lock file. Add an NPM script `setup` with implementation `npm ci`. Ask users to run `npm run setup`.

Reply:
done